core/target-systems/mongodb-springdata-target-system/src/main/java/io/flamingock/targetsystem/mongodb/springdata/MongoDBSpringDataTargetSystem.java

```diff
@@ -47,6 +51,7 @@ public class MongoDBSpringDataTargetSystem extends TransactionalTargetSystem<Mon
     public MongoDBSpringDataTargetSystem(String id, MongoTemplate mongoTemplate) {
         super(id);
         this.mongoTemplate = mongoTemplate;
+        this.databaseAccesor = new MongoDBSpringDataDatabaseAccesor(mongoTemplate);
```

Comment:
Why do we need the `MongoDBSpringDataDatabaseAccesor` at all? cannot we just return `mongotemplate.getDb();`?